docs/template_plugin/tests/functional/op_reference/acosh.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <vector>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+
+namespace {
+
+struct Tensor {
+    Tensor() = default;
+
+    Tensor(const ngraph::Shape& shape, ngraph::element::Type type, const InferenceEngine::Blob::Ptr& data): shape {shape}, type {type}, data {data} {}
+
+    template <typename T>
+    Tensor(const ngraph::Shape& shape, ngraph::element::Type type, const std::vector<T>& data_elements)
+        : Tensor {shape, type, CreateBlob(type, data_elements)} {}
+
+    ngraph::Shape shape;
+    ngraph::element::Type type;
+    InferenceEngine::Blob::Ptr data;
+};
+
+struct AcoshParams {
+    Tensor input;
+    Tensor expected;
+};
+
+struct Builder {
```

Comment:
Looks like it can be a common helper

Reply:
We should introduce some namespace for this test if we want to provide some tools. 
@ilyachur, do you have any suggestions?

Reply:
Should I move https://github.com/pelszkow/openvino/commit/a16ab8bdf1baf8c7ff83a830091bb79b89d7d759 to this PR or let create separate one?

Reply:
Yes I propose to move this change to this PR.
I think will be better to have such classes in common place. 
Or in other case we need to refactor this PR later.

Reply:
done